app/views/admin/petitions/edit.html.erb

```diff
@@ -2,7 +2,11 @@
 
   <%= render :partial => 'petition_details', :locals => { :f => f } %>
 
-  <div>
+  <div class="row">
+    <%= link_to 'Edit petition content', admin_petition_petition_details_path(@petition) %>
+  </div>
```

Comment:
You can put a guard clause on showing the link though, so it only appears for the petitions you're allowed to edit in this way (it's what I did for the debate outcomes link).